presto-docs/src/main/sphinx/security/hive.rst

```diff
@@ -0,0 +1,241 @@
+===========================
+Hive Security Configuration
+===========================
+
+The default security configuration of the :doc:`/connector/hive` uses simple
+authentication.  In this mode, the Hive connector connects to a Hadoop cluster
+that has been configured to use ``simple`` authentication. All queries are
+executed as the user who runs the Presto process, regardless of the user who
+submits the query.
+
+The Hive connector provides addional security options to support Hadoop
+clusters that have been configured to use :ref:`Kerberos
+<hive-security-kerberos-support>` for greater security.
+
+When accessing :abbr:`HDFS (Hadoop Distributed File System)`, Presto can
+:ref:`impersonate<hive-security-impersonation>` the end user who is running the
+query. This can be used with HDFS permissions and :abbr:`ACLs (Access Control
+Lists)` to provide additional security for data.
+
+.. _hive-security-kerberos-support:
+
+Kerberos Support
+================
+
+In order to use the Hive connector with a Hadoop cluster that uses ``kerberos``
+authentication, you will need to configure the connector to work with two
+services on the Hadoop cluster:
+
+* The Hive metastore Thrift service
+* The Hadoop Distributed File System (HDFS)
+
+Access to these services by the Hive connector is configured in the properties
+file that contains the general Hive connector configuration.
+
+Hive Metastore Thrift Service
+-----------------------------
+
+In a Kerberized Hadoop cluster, Presto accesses the Hive metastore Thrift
+service through :abbr:`SASL (Simple Authentication and Security Layer)`.
+Kerberos authentication for the metastore is configured in the connector's
+properties file using the following properties:
+
+================================================== ============================================================ ==========
+Property Name                                      Description                                                  Default
+================================================== ============================================================ ==========
+``hive.metastore.authentication.type``             One of ``SIMPLE`` or ``SASL``. When set to ``SASL`` the Hive ``SIMPLE``
+                                                   connector will connect to the Hive metastore Thrift service
+                                                   using SASL and authenticate with Kerberos. When using the
+                                                   default value of ``SIMPLE``, Kerberos authentication is
+                                                   disabled and no other properties need to be configured.
+
```

Comment:
In Description, would suggest putting the sentence "When using the default value of "SIMPLE" ...) right after the first sentence, since the rest has to do with SASL (and SIMPLE, then SASL, is also the order of the sections after this table).